src/core/server/app/router/api/index.ts

```diff
@@ -51,6 +53,8 @@ export function createAPIRouter(app: AppOptions, options: RouterOptions) {
   router.use("/account", createNewAccountRouter(app, options));
   router.use("/user", createNewUserRouter(app));
 
+  router.get("/oembed", oembedHandler());
```

Comment:
Capitalization must be consistent throughout the filename/exports. Rename files/folders down from `api/oEmbed/oEmbed` to `api/oembed/oembed`.